### PR TITLE
Fr - grammar.xml : False positive corrections in PLACE_DE_LA_VIRGULE rule

### PR DESCRIPTION
**Pull request by OswaldLuc**
French - False positive corrections in file grammar.xml by adding antipatterns in PLACE_DE_LA_VIRGULE rule group.

### CODE REVIEW

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 1]
```diff
@@ -25733,17 +25792,6 @@ though, and has since been slightly modified)-->
...
-            <!-- simplified version of FAIRE_VPPA2 -->
-            <pattern>
-                <token postag="V.*" postag_regexp="yes" inflected="yes">voir</token>
-                <marker>
-                    <token postag="V.* ppa.*" postag_regexp="yes" regexp="yes">.*ée?s?</token>
-                </marker>
-            </pattern>
-            <message>Vouliez-vous dire  <suggestion><match no="2" postag="V.*" postag_regexp="yes" postag_replace="V.* inf"/></suggestion> (infinitif)?</message>
-            <example correction="passer">J’ai refusé des sponsors avant de les voir <marker>passé</marker>.</example>
-        </rule>
```
Comment (udomai) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:25746]: Did you remove this rule on purpose?

**Review by udomai — CHANGES_REQUESTED** [review]

**Comment by udomai** [comment]
@Rebecca-auque, could you check if you agree with the changes in PLACE_DE_LA_VIRGULE?

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml (hunk 1 above)
Comment (jaumeortola) [reply languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:25746]: The removed rule and the added rule D_N_J_VPPA (removed by me yesterday) come from an older version. This should not happen. I don't know the reason.

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 2]
```diff
@@ -4042,6 +4042,65 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
             <example>Ces pu(ri)tains de Suisses ils ont peur de voir LES CANICULES S'EMBALLER !</example>
             <example>La république a dans les faits duré peu de temps.</example>
         </rule>
+        <rule id="D_N_J_VPPA" name="L'expérience journalistique engagé (engagée)" default="temp_off">
```
Comment (udomai) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:4045]: Nice! We actually don't find the corrected example yet 🤓

I've found FPs in the rule editor:

> Étienne Davignon, à l'époque jeune **attaché** aux affaires étrangères.
> Le discours de bienvenue à la délégation soviétique **prononcé** par Gustáv Husák est parfaitement en ligne avec le contenu du traité.
> En 1844, il a publié un essai défendant la prosodie romantique intitulé Prosodie de l'école moderne, précédé d'une lettre de Victor Hugo et préfacé par Émile Deschamps.

Possible pitfalls are nouns that look like `V ppa.*` and more complex attribute structures (with `V ppr.*` for example).

**Review by udomai — CHANGES_REQUESTED** [review]

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml (hunk 1 above)
Comment (udomai) [reply languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:25746]: Then my other commentary is irrelevant, deleting :)

**Comment by Rebecca-auque** [comment]
I don't agree with  <example>Aussi<marker> se tint-il</marker> prudemment entre deux eaux.</example>. This sentence needs a comma and I don't understand why the <token postag="A.* inte" postag_regexp="yes" skip="-1"/> was taken out in one AP of the same subrule. The other changes seem relevant.

**Comment by jaumeortola** [comment]
Thanks, @OswaldLuc. Your changes have been added partially here: https://github.com/languagetool-org/languagetool/pull/5603

**Comment by OswaldLuc** [comment]
Hi,

I do not agree with you about the sentence :

/Aussi se tint-il prudemment entre deux eaux.

/ In this sentence, a comma would be a mistake, according to the rule 
saying that "Aussi" at the beginning of a sentence is followed by a 
comma, except when the subject is post-pone.

This rule can be easily found and explained with examples in numerous 
french grammar websites, e.g. :

https://usito.usherbrooke.ca/articles/aides_%C3%A0_la_r%C3%A9daction/LesPrincipalesR%C3%A8glesDEmploiDeLaPonctuation 
<https://usito.usherbrooke.ca/articles/aides_%C3%A0_la_r%C3%A9daction/LesPrincipalesR%C3%A8glesDEmploiDeLaPonctuation>

https://www.etudes-litteraires.com/forum/discussion/45382/virgule-apres-un-adverbe-place-en-debut-de-phrase 
<https://www.etudes-litteraires.com/forum/discussion/45382/virgule-apres-un-adverbe-place-en-debut-de-phrase>

https://www.btb.termiumplus.gc.ca/redac-chap?lang=eng&lettr=chapsect6&info0=6.1.17 
<https://www.btb.termiumplus.gc.ca/redac-chap?lang=eng&lettr=chapsect6&info0=6.1.17>

https://www.btb.termiumplus.gc.ca/redac-chap?lang=fra&lettr=chapsect6&info0=6.1 
<https://www.btb.termiumplus.gc.ca/redac-chap?lang=fra&lettr=chapsect6&info0=6.1>

https://forum.wordreference.com/threads/fr-aussi-virgule.3350481/ 
<https://forum.wordreference.com/threads/fr-aussi-virgule.3350481/>

https://www.langue-fr.net/Aussi-Ainsi-difference-et-construction 
<https://www.langue-fr.net/Aussi-Ainsi-difference-et-construction>








Le 31/08/2021 à 13:49, Rebecca-auque a écrit :

**Comment by jaumeortola** [comment]
Thank you, @OswaldLuc. It is fixed now here: https://github.com/languagetool-org/languagetool/pull/5620

**Pull request closed without merging**